src/audio/__tests__/AudioProvider.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { fireEvent, render } from '@testing-library/react';
+import { RecoilRoot } from 'recoil';
+import useAudio from '../__mocks__/use-audio.ts';
+import { AudioProvider } from '../AudioProvider';
+
+vi.mock('../__mocks__/useAudio.ts');
+
+const AudioController = () => {
+  const audio = useAudio();
+
+  return (
+    <div>
+      <button aria-label="Play" onClick={audio.playAudio}></button>
+      <button aria-label="Stop" onClick={audio.stopAudio}></button>
+      <button aria-label="Pause" onClick={audio.pauseAudio}></button>
+    </div>
+  );
+};
+
+const setup = () =>
+  render(
+    <RecoilRoot>
+      <AudioProvider>
```

Comment:
You should not need these if mocks are properly applied

Reply:
test is updated to match this requirement